CRM/Case/Form/Case.php

```diff
@@ -305,8 +315,6 @@ public static function formRule($values, $files, $form) {
    * Process the form submission.
    */
   public function postProcess() {
-    $transaction = new CRM_Core_Transaction();
-
```

Comment:
Are you sure this is safe to remove? I'm not exactly sure how transactions work but once you create one it does stuff even if you don't reference the variable, right?

Reply:
@colemanw I think you are correct.  I just saw an "unused" variable in phpstorm and removed the line.  I've now "unremoved" it in this PR.